clients/src/main/java/org/apache/kafka/common/requests/ListOffsetRequest.java

```diff
@@ -184,7 +177,6 @@ public String toString() {
             if (partitionTimestamps != null) {
                 bld.append(", partitionTimestamps=").append(partitionTimestamps);
             }
-            bld.append(", minVersion=").append(minVersion);
```

Comment:
Is this information not useful?

Reply:
The value seemed a little dubious to me since it can be inferred from the rest of the `toString` (after I add the missing isolation level) . We should get clear detail from the `UnsupportedVersionException` if we hit one, so it's just noise otherwise. That said, there's no particular harm having it.